dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/patch/factories/impl/DspaceObjectMetadataPatchUtils.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.repository.patch.factories.impl;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.exception.DSpaceBadRequestException;
+import org.dspace.app.rest.model.MetadataValueRest;
+import org.dspace.app.rest.model.patch.JsonValueEvaluator;
+import org.dspace.app.rest.model.patch.Operation;
+import org.dspace.content.MetadataField;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.MetadataFieldService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Util class for shared methods between the Metadata Operations
+ * @author Maria Verdonck (Atmire) on 18/11/2019
+ */
+@Component
+public final class DspaceObjectMetadataPatchUtils {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    @Autowired
+    private MetadataFieldService metadataFieldService;
+
+    /**
+     * Path in json body of patch that uses this operation
+     */
+    protected static final String METADATA_PATH = "/metadata";
+
+    private DspaceObjectMetadataPatchUtils() {
+    }
+
+    /**
+     * Extract metadataValue from Operation by parsing the json and mapping it to a MetadataValueRest
+     * @param operation     Operation whose value is begin parsed
+     * @return MetadataValueRest extracted from json in operation value
+     */
+    protected MetadataValueRest extractMetadataValueFromOperation(Operation operation) {
+        MetadataValueRest metadataValue = null;
+        try {
+            if (operation.getValue() != null) {
+                if (operation.getValue() instanceof JsonValueEvaluator) {
+                    JsonNode valueNode = ((JsonValueEvaluator) operation.getValue()).getValueNode();
```

Comment:
I see that you are not using the `JsonValueEvaluator.evaluate()` implementation to get the node value and that you're letting `ObjectMapper` throw the exception. I think that has the desired effect, but just want to share that `JasonValueEvaluator` throws a Spring `PatchException`, which might be worth considering here.